docs/search/index.asciidoc

```diff
@@ -19,53 +19,53 @@ The *Search* space in {kib} comprises the following features:
 
 The Search solution and use case is made up of many tools and features across the {stack}.
 As a result, the release notes for your features of interest might live in different Elastic docs.
-// Use the following table to find links to the appropriate documentation, API references (if applicable), and release notes.
+Use the following table to find links to the appropriate documentation, API references (if applicable), and release notes.
 
-// [options="header"]
-// |===
-// | Name               | API reference           | Documentation         | Release notes
+[options="header"]
+|===
+| Name               | API reference           | Documentation         | Release notes
 
-// | Connectors
-// | link:https://example.com/connectors/api[API reference]
-// | link:https://example.com/connectors/docs[Documentation]
-// | link:https://example.com/connectors/notes[Release notes]
+| Connectors
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/8.15/connector-apis.html[API reference]
+| <<action-types,Documentation>>
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/es-release-notes.html[Elasticsearch guide]
 
-// | Web crawler
-// | link:https://example.com/web_crawlers/api[API reference]
-// | link:https://example.com/web_crawlers/docs[Documentation]
-// | link:https://example.com/web_crawlers/notes[Release notes]
+| Web crawler
+| link:https://www.elastic.co/guide/en/app-search/current/web-crawler-api-reference.html[API reference]
+| link:https://www.elastic.co/guide/en/enterprise-search/current/crawler.html[Documentation]
+| link:https://www.elastic.co/guide/en/enterprise-search/current/changelog.html[Enterprise Search Guide]
```

Comment:
```suggestion
| N/A
| {enterprise-search-ref}/crawler.html[Documentation]
| {enterprise-search-ref}/changelog.html[Enterprise Search Guide]
```

Reply:
The Elastic web crawler doesn't have API access. But the previous App Search crawler did 🤯!